_posts/2016-11-12-TheServerlessCloud.md

```diff
@@ -0,0 +1,230 @@
+---
+layout: post
+authors: [bart_blommaerts]
+title: 'The Serverless Cloud'
+image: /img/serverless.jpg
+tags: [Cloud, Serverless, AWS, Lambda, Azure, Google Cloud Platform, IBM Bluemix, OpenWhisk, Webtask]
+category: Cloud
+comments: true
+---
+
+>In recent years, the uprise of the cloud has brought us a lot of new and disruptive technologies. 
+Everybody is talking about SaaS, PaaS, IaaS and other sorts of aaS. 
+In 2014, Amazon launched AWS Lambda as the pinnacle of the cloud computing. 
+It allows developers to focus on code, without spending time on managing servers.
+
+# Part 1
+
+## What?
+
+While Microservices have been reigning the Cloud landscape for a couple of years, today the Serverless movement is one of the hottest trends in the industry. 
+Historically, software developers have been pretty bad at naming things and Serverless is no exception. 
+Disregarding what the name suggests, Serverless does not imply the complete absence of servers. 
+It implies that developers who are using the Serverless architectural style, are not responsible for managing or provisioning the servers themselves, but use a vendor-supplied Cloud solution. 
+Serverless means less worrying about servers. 
+Although in the future, it might be possible to install this kind of service on-premise, for example with the open-source [IBM OpenWhisk](https://github.com/openwhisk/openwhisk) implementation.
+
+In regard to this, the definition [FaaS](https://twitter.com/marak/status/736357543598002176): Functions as a Service makes a lot more sense. 
+Functions are short-lived pieces of runtime functionality that don’t need a server that’s always running. 
+Strictly speaking a function can have a longer execution time, but most FaaS providers will currently limit the allowed computation time. 
+When an application calls a function (eg. a calculation algorithm) this function gets instantiated on request. 
+After finishing it gets destroyed. 
```

Comment:
Maybe rephrase to make it sound better?
**When it is finished,** it gets destroyed.